### PR TITLE
Adding changes to readme and configuration files based on edit report configuration. 

### LINKED ISSUE #703: Update documentation for the edit report configuration.

### PR DESCRIPTION
**Pull request by MichaelFurey17**
Close #703

### CODE REVIEW

File: 2018/python/file_generator.py [hunk 1]
```diff
@@ -468,10 +468,15 @@ def edit_report(self):
 				getattr(checker, func)()
 
 		#Creates a dataframe of results from the checker. 
-		res_df = pd.DataFrame(checker.results)
+		report_df = pd.DataFrame(checker.results)
 
-		#Filters the results for edits that have failed. 
-		res_df = res_df[(res_df['status']=='failed')]
+		#Creating a report based on the status of the edits failed.
+		if self.edit_report_config['status'] == 'failed':
```
Comment (Kibrael) [inline 2018/python/file_generator.py]: Is there a reason to not combine all the edit results into a single report?

It seems like having a single report would simplify things and one could use the status in the report to filter for needed results.

File: readme.md [hunk 2]
```diff
@@ -87,31 +87,62 @@ Large files are used during load testing. Large test files are created using a c
...
-- output_filename: `large_file_10000_rows.txt`
-- *`row_by_row_modification_yaml_file`:
-    * The script contains functionality for changing values by column and row using the [2018 yaml configuration](https://github.com/cfpb/hmda-test-files/blob/master/2018/python/configurations/row_by_row_modification.yaml) or the [2019 yaml configuration](https://github.com/cfpb/hmda-test-files/blob/master/2019/python/configurations/row_by_row_modification.yaml). The path to the row by row configuration file can be added here to implement the data changing functionality. There is no default value.
-- bank_name: `Bank1`
-- lei: `BANK1LEIFORTEST12345`
-- tax_id: `02-1234567`
-- row_count: `10000`
+Configuration options include (with defaulted values):
+
+|Key|Default Value|Data Type|Expanation|
```
Comment (Kibrael) [inline readme.md]: Typo in Explanation

File: readme.md [hunk 3]
```diff
@@ -87,31 +87,62 @@ Large files are used during load testing. Large test files are created using a c
  [2019 Large File Configuration](https://github.com/cfpb/hmda-test-files/tree/master/2019/configurations/test_filepaths.yaml).
```
Comment (Kibrael) [inline readme.md]: Can you drop these two files down into a bulleted list?

**Comment by Kibrael** [comment]
Looks like this one picked up some merge conflicts due to other PRs. Other than that LGTM.

File: 2018/python/file_generator.py (hunk 1 above)
Comment (MichaelFurey17) [reply 2018/python/file_generator.py]: Sure - I think that would be fine to output a full edit report.

File: readme.md (hunk 2 above)
Comment (MichaelFurey17) [reply readme.md]: Thanks for the catch!

File: readme.md (hunk 3 above)
Comment (MichaelFurey17) [reply readme.md]: Yep!

**Comment by MichaelFurey17** [comment]
Thanks, David! I will merge.

**Pull request merged**